algorithms/finetune/iql.py

```diff
@@ -485,7 +488,13 @@ def _update_q(
         # Update target Q network
         soft_update(self.q_target, self.qf, self.tau)
 
-    def _update_policy(self, adv, observations, actions, log_dict):
+    def _update_policy(
+        self,
+        adv: torch.Tensor,
+        observations: torch.Tensor,
+        actions: torch.Tensor,
+        log_dict: Dict,
```

Comment:
shouldn't we specify [str, Any/Whatever?]

Reply:
Idk, for me that seems fine